test/integration/traces_test.go

```diff
@@ -105,6 +125,54 @@ func testHTTPTraces(t *testing.T) {
 	}), "not all tags matched in %+v", process.Tags)
 }
 
+func testHTTPTracesBadTraceID(t *testing.T) {
+	traceID := createTraceID()
+	slugToParent := map[string]string{
+		// Valid traceparent example:
+		//		valid: "00-5fe865607da112abd799ea8108c38bcb-4c59e9a913c480a3-01"
+		//      valid: "00-"+traceID+"-4c59e9a913c480a3-01"
+		// Examples of INVALID traceparents:
```

Comment:
This is great! Perhaps we can add some unit tests on the extraction code as well, especially strings less than 3 characters long.

Reply:
Extract code should no longer need anything tested.  EBPF only copies the data when its length is 55, and we will not use values that do not pass checks in otel trace's TraceIDFromHex.